pkg/controller/kieapp/constants/constants.go

```diff
@@ -114,7 +114,7 @@ const (
 
 	DatagridVar         = "DATAGRID_IMAGE_"
 	Datagrid73Image     = "datagrid73-openshift"
-	Datagrid73ImageTag  = "1.3"
+	Datagrid73ImageTag  = "1.5"
```

Comment:
i assume you only want this change to affect release 7.8? if so, you'll need to mess with `VersionConstants` a bit in this same file... maybe you need a new constant for this datagrid tag?

Reply:
7.8 and 7.7.1
for 7.7.1 I assume I'll have to update it on 1.4.x branch, yes?

Reply:
right, so that means for this branch... you'll need to do as i said to ensure 7.7.0 release isn't affected.